iquip/apps/dataviewer.py

```diff
@@ -691,6 +691,7 @@ def _idToggledSlot(self, id_: int, checked: bool):
             checked: Whether the button is now checked or not.
         """
         if checked:
+            self.buttonGroup.button(id_).setChecked(True)
```

Comment:
What is this for?

Reply:
I thought I needed this when I was testing before, but when I looked back, I don't think so.
Please ignore it😅